### DISCUSSION TITLE
join tables with count

### DISCUSSION THREAD

**Discussion by ricardosikic** [supabase/supabase#9284] [Questions] [answered]
I was doing a query that returns the number of participants per event, currently I handle that with `rpc()`, but I would like to know how to do a count of rows with a join directly without doing `rpc()`. What I do now is do the join and from that table I call the id but I can't count it, if not I count it separately with the array that returns me. My actual query is below, `event_participants` is the array i want to count but at the same query

```
      .from('events')
      .select(`
      id,
      title,
      description,
      latitude,
      longitude,
      creator,
      start_date,
      event_participants(
        id
      )
      `)
```

**Comment by GaryAustin1** [comment, marked as answer]
This might help:
https://github.com/supabase/supabase/discussions/1113#discussioncomment-594673

**Reply by ricardosikic** [reply]
Thanks @GaryAustin1
